service.go

```diff
@@ -84,6 +85,23 @@ func (service *Service) HasTool(category ToolCategory, name string, environment
 	return false
 }
 
+func (service *Service) GetSystem(client *Client) (*Identifier, error) {
```

Comment:
I'm curious about this since it doesn't fit the pattern we see in the other `Service` member functions.
All "GetResource" member functions return a connection type, which lines up with the public grahpql schema. I checked and it looks like we're getting the Identifier data of `Parent` just by adding the field to the struct.

Happy to discuss this on a call if that's better 😄

Reply:
<img width="458" alt="image" src="https://github.com/OpsLevel/opslevel-go/assets/37668804/9080b0c8-358f-483e-978d-8bb29867b273">

@davidbloss `Parent` is just a nullable foreign key unlike properties and repositories. Just life `Lifecycle`.